sdk/keyvault/azure-security-keyvault-common/src/keyvault_exception.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we need file at all or its .hpp file.

Reply:
Will go away eventually once this exception (and the one from Storage) are combined and moved to Core